### DISCUSSION TITLE
Mix line and texture gsl ?

### DISCUSSION THREAD

**Discussion by Airioo** [bnpr/Malt#78] [Q&A] [answered]
Hello !

I have another question : is this possible to have a line.gsl with a texture.gsl ? 
Whitout using solidify modifyer ?

**Comment by pragma37** [comment, marked as answer]
Sure! Just take a look at the code of both shaders and try to combine them.

It shouldn't be too hard even if you don't have any programming experience.
[This tutorial](https://malt3d.com/#/docs/From-Nodes-To-Code/From-Nodes-To-Code) can help you to understand the basics.

**Reply by Airioo** [reply]
Nice thanks !
